extensions/opentelemetry/opentelemetry/deployment/src/test/java/io/quarkus/opentelemetry/deployment/TestUtil.java

```diff
@@ -0,0 +1,64 @@
+package io.quarkus.opentelemetry.deployment;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.trace.TracerProvider;
+import io.opentelemetry.context.propagation.TextMapPropagator;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.sdk.trace.IdGenerator;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.samplers.Sampler;
+import io.quarkus.arc.Unremovable;
+
+@Unremovable
+public final class TestUtil {
```

Comment:
Created this class to be able to properly check if we are setting the right values... the down side of it is that it is using reflection since everything there is either private field, or unexported class....

Reply:
Not ideal, but for tests, I think it's ok.